### DISCUSSION TITLE
actions not supported by Vorta i/f

### DISCUSSION THREAD

**Discussion by rw227** [borgbase/vorta#1441] [FAQ]
Hi,

If I want to run a borg command not supported in Vorta i/f, where is the borg app stored or how do I call it?

Thank you

**Comment by real-yfprojects** [comment]
On which OS are you and where did you install vorta from?

**Reply by rw227** [reply]
running on LinuxMint and installed Flatpak version ... Vorta v0.8.7, Borg v1.2.0. Borg was not previously installed on system.

Thank you.

**Reply by real-yfprojects** [reply]
The easiest approach is [installing borg](https://borgbackup.readthedocs.io/en/stable/installation.html) normally on your system.

If you absolutely want to use the borg executable bundled with vorta, you can do so the following way:
```
flatpak run --command=borg com.borgbase.Vorta <arg1> <arg2>
```

**Reply by rw227** [reply]
Thank you, perfect. I did not want to have to install borg normally for the odd occasion I needed to do something off-Vorta. 
regards
